cross/cops/Makefile

```diff
@@ -1,5 +1,5 @@
 PKG_NAME = cops
-PKG_VERS = 1.5.4
+PKG_VERS = 3.2.2
```

Comment:
This is a large step
There is version 2.8.2 that is still active.
If COPS v2.x works with DSM 6 I propose to update to the latest 2.x version
And for COPS v3 we could create a new package (cops3 ?) for DSM 7 only

Reply:
Hmm, i'm not seeing what you refer to. Based on the release notes for [2.0.1 Initial release for PHP >= 8.1 with new EPub update package](https://github.com/mikespub-org/seblucas-cops/releases/tag/2.0.1) it shows this:

> The 2.x.x releases will support PHP >= 8.1 which is typical for most source code & docker image installs now.
> The 1.x.x releases will be the only ones supporting PHP >= 7.4 in the future.

If this is the case based on the Synology [Release Notes for PHP 8.1](https://www.synology.com/en-us/releaseNote/PHP8.1):
> (2023-06-15)
> **Compatibility & Installation**
>
> PHP 8.1.18-0102 requires DSM 7.2 and above.
> **What's New**
>
> This is the initial release of package [PHP 8.1](https://www.php.net/ChangeLog-8.php#PHP_8_1).

As such, once you go beyond version 1.x.x

Reply:
@hgy59 is this good to merge?